### PR TITLE
#19: Update spack to look for magistrate

### LINKED ISSUE #18: Test spack install
We don't have any testing of spack

### LINKED ISSUE #19: Update spack to look for magistrate

### PR DESCRIPTION
**Pull request by JacobDomagala**
Fixes #19 Fixes #18

### CODE REVIEW

**Review by nmm0 — APPROVED** [review]
Overall I think this looks good, but we may want backwards compatibility with older Darma versions? I suppose it depends on if anyone is actually using these packages

**Comment by JacobDomagala** [comment]
Moving this to draft. Tested it locally and it still fails

**Comment by JacobDomagala** [comment]
Ok, works now. I also added CI that runs `spack`

File: ci/test_spack.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```
Comment (cz4rs) [inline ci/test_spack.sh:1]: This feels like a lot of bash code to achieve something that could be done more concisely in the workflow file :thinking:

Comment (JacobDomagala) [reply ci/test_spack.sh:1]: Having this as a script file allows for local testing as well

Comment (cz4rs) [reply ci/test_spack.sh:1]: Yeah, I think that's a strong enough argument.

**Review by cz4rs — APPROVED** [review]

**Review by lifflander — APPROVED** [review]

**Pull request merged**